.changelog/318.feature.2.md

```diff
@@ -0,0 +1 @@
+docker: Update mainnet genesis
```

Comment:
@pro-wh, I've added a copy of the Change Log fragment with the original PR number so it will be linked to both PRs in the final generated Change Log.

Reply:
oh that's better, thanks